oem/firstvoices/keyboards.csv

```diff
@@ -52,8 +52,10 @@ fv,fv_ilnu_innu_aimun,Ilnu-Innu Aimun,Eastern Subarctic,fv_ilnu_innu_aimun_kmw-9
 fv,fv_swampy_cree,ᐃᓂᓂᒧᐎᐣ (Swampy Cree),Eastern Subarctic,fv_swampy_cree_kmw-9.0.js,9.2.1,csw-Cans,Swampy Cree (Unified Canadian Aboriginal Syllabics)
 fv,fv_moose_cree,ᐃᓕᓖᒧᐎᓐ (Moose Cree),Eastern Subarctic,fv_moose_cree_kmw-9.0.js,9.2.1,crm-Cans,Moose Cree (Unified Canadian Aboriginal Syllabics)
 fv,fv_northern_east_cree,ᐄᔨᔫ-ᐄᓅ ᐊᔨᒨᓐ (Northern East Cree),Eastern Subarctic,fv_northern_east_cree_kmw-9.0.js,9.2.1,crl-Cans,Northern East Cree (Unified Canadian Aboriginal Syllabics)
-fv,fv_severn_ojibwa,ᐊᓂᔑᓂᓂᒧᐎᐣ (Severn Ojibwa),Eastern Subarctic,fv_severn_ojibwa_kmw-9.0.js,9.3.1,ojs-Cans,Severn Ojibwa (Unified Canadian Aboriginal Syllabics)
-fv,fv_ojibwa,ᐊᓂᔑᓇᐯᒧᐎᓐ (Ojibwa),Eastern Subarctic,fv_ojibwa_kmw-9.0.js,9.3.1,ojb-Cans,Northwestern Ojibwa (Unified Canadian Aboriginal Syllabics)
+fv,fv_severn_ojibwa,ᐊᓂᔑᓂᓂᒧᐎᐣ (Severn Ojibwa),Eastern Subarctic,fv_severn_ojibwa_kmw-9.0.js,10.0.1,ojs-Cans,Severn Ojibwa (Unified Canadian Aboriginal Syllabics)
+fv,fv_severn_ojibwa_rdot,ᐊᓂᔑᓂᓂᒧᐏᐣ (Severn Ojibwa right w-dot),Eastern Subarctic,fv_severn_ojibwa_kmw-9.0.js,1.0.1,ojs,Oji-Cree
+fv,fv_ojibwa,ᐊᓂᔑᓇᐯᒧᐎᓐ (Ojibwa),Eastern Subarctic,fv_ojibwa_kmw-9.0.js,10.0.1,ojb-Cans,Northwestern Ojibwa (Unified Canadian Aboriginal Syllabics)
+fv,fv_ojibwa_rdot,ᐁᓂᔑᓇᐯᒧᐏᓐ (Ojibwa right w-dot),Eastern Subarctic,fv_ojibwa_rdot_kmw-9.0.js,1.0.1,oj,Ojibwa
```

Comment:
@HopsAndHops - Can you review eab77b6d4e912d37c21dd0119d528a02829a76ea and provide guidance what the rdot keyboard names should display as? They're quite long, and I already truncated a little to:
ᐁᓂᔑᓇᐯᒧᐏᓐ (Ojibwa right w-dot)
ᐊᓂᔑᓂᓂᒧᐏᐣ (Severn Ojibwa right w-dot)

For the rdot keyboards, I used the language information from the .kps files, which differ slightly. Let me know what it should be.

Reply:
Those titles all look okay to me! As a heads up, I do have 2 more variants to the keyboard to be PR'd. I'll think about a shorter title for them and tag you with them when I make those PRs.

Reply:
Do you think it would make sense differentiate the a- and i-finals keyboards like this?:

ᐊᓂᔑᓇᐯᒧᐎᓐ (a-finals)
ᐁᓂᔑᓇᐯᒧᐏᓐ (a-finals right w-dot)
ᐊᓂᔑᓇᐯᒧᐎᣙ (i-finals)
ᐊᓂᔑᓇᐯᒧᐏᣙ (i-finals, right w-dot)

Reply:
Yep - could you list those names for me in the following order?

fv_severn_ojibwa
fv_severn_ojibwa_rdot
fv_ojibwa
fv_ojibwa_rdot

Reply:
> Do you think it would make sense to differentiate the a- and i-finals keyboards like this?:
> 
> ᐊᓂᔑᓇᐯᒧᐎᣙ (i-finals) 
> ᐊᓂᔑᓇᐯᒧᐏᣙ (i-finals, right w-dot)


Are i-finals for an upcoming keyboard PR? 

I saw the "Keyboard Details" for 
https://keyman.com/keyboards/fv_ojibwa_rdot

mention [ᐊᓂᔑᓇᐯᒧᐏᣙ (Ojibwa, i-finals, right w-dot](https://www.keyman.com/keyboards/fv_ojibwa_ifinal_rdot) but I don't see that keyboard exists.